app/src/main/java/com/project/symptoms/activity/BodySelection.java

```diff
@@ -41,7 +41,6 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
 
         setButtonsListeners();
 
-        initialDBInsertion();
```

Comment:
this is moved from here because it may cause multiple insertions if the user goes back from main activity so this OnCreate is called again though it will jump to MainActivity again